aces/Loris
# [Media] Data FrameWork Implementation

### Brief summary of changes
Implemented the Data Framework for the Media Module. 

### To Test
- Make sure that everything is loading properly in the datatable
- Make sure the filter is filtering data properly
- Make sure that when the permissions are not set to 'superuser', the hidden files are indeed hidden from the datatable.
